pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -115,11 +122,66 @@ func (r *Cluster) ValidateUpdate(old runtime.Object) error {
 	return nil
 }
 
+// ValidateEksaVersionSkew ensures that upgrades are sequential by CLI minor versions.
+func ValidateEksaVersionSkew(new, oldCluster *Cluster) field.ErrorList {
+	var allErrs field.ErrorList
+	eksaVersionPath := field.NewPath("spec").Child("EksaVersion")
+
+	if new.Spec.EksaVersion == nil || oldCluster.Spec.EksaVersion == nil {
+		return nil
+	}
+
+	oldEksaVersion, err := semver.New(string(*oldCluster.Spec.EksaVersion))
+	if err != nil {
+		allErrs = append(
+			allErrs,
+			field.Invalid(eksaVersionPath, new.Spec.EksaVersion, "old cluster is invalid"))
```

Comment:
```suggestion
			field.Invalid(eksaVersionPath, old.Spec.EksaVersion, "old cluster is invalid"))
```

However, should we error out here? If the old cluster is invalid, the user will never be able to update it.

Reply:
I don't think it should error out.